app/controllers/partners_controller.rb

```diff
@@ -103,7 +114,7 @@ def autovivifying_hash
   end
 
   def partner_params
-    params.require(:partner).permit(:name, :email, :send_reminders)
+    params.require(:partner).permit(:name, :email, :send_reminders, :status)
```

Comment:
Since you've created a controller action for this, I don't think you have to whitelist the field in the params